doc/admin/config.rst

```diff
@@ -347,6 +360,22 @@ to speed up various operations::
     If your redis setup doesn't require a password or you already specified it in the location you can omit this option.
     If this is set it will be passed to redis as the connection option PASSWORD.
 
+``ssl_cert_reqs``
+    If your redis setup doesn't require TLS or mTLS you can omit this option.
+    If this is set it will be passed to redis as the connection option ``SSL_CERT_REQS``. Possible values are none, optional, required.
```

Comment:
```suggestion
    If this is set it will be passed to redis as the connection option ``SSL_CERT_REQS``. Possible values are ``none``, ``optional``, and ``required``.
```

Reply:
Done